content/analyst/inbox-and-design-tracker/how-to-create-an-abuse.net-listing.md

```diff
@@ -0,0 +1,25 @@
+---
+lastUpdated: "02/23/2023"
+title: How to Create an Abuse.net Listing
+description: Don't have an Abuse.net listing? Here's how to create one
+---
+
+
+ An [abuse.net](http://abuse.net) listing is a best practice anyone who sends promotional email. This record is managed by *Network Clearing House*, a public database that keeps track of the main contacts for various sending domains. If a deliverability issue occurs, this company will ensure the right person is notified so the problem can be resolved.
+
+**How to Create or Update a Listing** The [abuse.net](http://abuse.net) contact database has about 200,000 domains, which is less than 1% of the all the domains that exist. If an entry is nonexistent or incorrect, Network Clearing House relies on mailers like us to submit up-to-date entries for their database. To do so, follow these steps:
+
+1. **Draft a plain text message to** [**update@abuse.net.**](mailto:update@abuse.net) Changing your mailing program to send plain text rather than formatted or HTML mail will allow the [abuse.net](http://abuse.net) system to decode your message.
+2. **Your message format** should use the following format: name of the domain, a colon, and the reporting address(es), all on one line separated by spaces.
+
+![](media/how_to_create_an_abusenet_listing/image_0.png)
+
+ You can send information for as many domains as you want in a single message. If there's more than one contact address for a domain, then send copies of mail to all of them. Don't use commas or mailto: URLs or any other extra punctuation. The entry for a domain automatically includes subdomains, so [mail.example.com](http://mail.example.com/) is the same as for [example.com](http://example.com/); separate entries aren't required.
+ 
+ 3. **Specify your entry.** Are these domains you are responsible for, or ones that you've researched? If the connection between a domain you've researched and the contact isn't obvious, explain how you found it.
```

Comment:
For some reason this is being rendered as a new list 
![image](https://user-images.githubusercontent.com/10710273/222256054-5c5f2b60-09c5-40e7-b427-ec7f65e851ca.png)